controller/controller.go

```diff
@@ -159,6 +161,16 @@ func New(config Config) (*Controller, error) {
 		}
 	}
 
+	var eventRecorder recorder.Interface
+	{
+		c := recorder.Config{
+			Component: config.Name,
+			K8sClient: config.K8sClient,
```

Comment:
`Component` is a setting and `K8sClient` is a dependency. See https://github.com/giantswarm/fmt/blob/master/go/struct_initialization.md. 

```suggestion
			K8sClient: config.K8sClient,

			Component: config.Name,
```